dev_env/java_nginx_env/Dockerfile

```diff
@@ -0,0 +1,26 @@
+# This is the default base image for use with user models and workflows.
+FROM datarobot/java-dropin-env-base
+
+RUN apt-get update && apt-get install -y vim nginx
```

Comment:
any reason for vim?

Reply:
these are dev envs, so just for debugging